### PR TITLE
KYP-827: magento2 default newsletter subscribe form handle

### PR DESCRIPTION
**Pull request by nedick007**
Simple modification that will allow the capture of guest customers subscribing to newsletter.

### CODE REVIEW

**Comment by nedick007** [comment]
@evgeni-spasov-metrilo ready for review

File: Observer/Customer.php [hunk 1]
```diff
@@ -43,8 +43,20 @@ private function getCustomerFromEvent($observer) {
...
+                } else {
+                    $subscriberEmail = $subscriber->getEmail();
+                    return new MetriloCustomer(
+                        $subscriber->getStoreId(),
+                        $subscriberEmail,
+                        strtotime($subscriber->getData('change_status_at')) * 1000,
+                        $subscriberEmail,
+                        $subscriberEmail,
+                        true,
+                        ['guestCustomer']
```
Comment (evgeni-spasov-metrilo) [inline Observer/Customer.php]: if this is the name of a tag, are you sure we use camel case for those?

Comment (nedick007) [reply Observer/Customer.php]: Tested this on production and there is no problem with case formatting, but API doc's are using snake case. Should I change it to 'guest_customer' ?

Comment (evgeni-spasov-metrilo) [reply Observer/Customer.php]: yup.

Comment (nedick007) [reply Observer/Customer.php]: Done.

**Comment by nedick007** [comment]
@evgeni-spasov-metrilo bump

**Pull request merged**